### PR TITLE
add package ecosystems blog post and author info

### PR DESCRIPTION
**Pull request by thoward**
## Description

- Add post about package ecosystems (Fixes https://github.com/pulumi/devrel-team/issues/938)
- Add info for new author

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/blog/next-level-iac-package-ecosystems/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+```typescript
+import * as mime from 'mime-types'
+
+const mimeType = mime.lookup(‘path/to/your/local/file.ext’);
+
+```
+
+So, to work that into our Pulumi program from above, we only need to make two small edits:
+
+```typescript {hl_lines=[4,21]}
```
Comment (cnunciato) [inline themes/default/content/blog/next-level-iac-package-ecosystems/index.md:84]: I always forget we can do this! The highlighting does seem to be making the code a bit hard to read, though, and for me at least, interrupting with the rest of the syntax highlighting that follows it. Not something that need block this PR necessarily, but would be great to get something filed to handle line highlighting more beatifully!

**Review by cnunciato — COMMENTED** [review]
Nice! Some initial feedback for ya. ⭐

File: themes/default/content/blog/next-level-iac-package-ecosystems/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Next-level IaC: Powering Up Pulumi with Package Ecosystems"
+allow_long_title: true
+date: 2024-05-08
```
Comment (interurban) [inline themes/default/content/blog/next-level-iac-package-ecosystems/index.md:4]: hey @thoward ; you'll want to modify the date for Monday our new target publish date

**Review by interurban — APPROVED** [review]
Dropping in an approval so you can merge when your fixes are in place.

File: themes/default/content/blog/next-level-iac-package-ecosystems/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Next-level IaC: Powering Up Pulumi with Package Ecosystems"
```
[the remark this answers is not in the document]
Comment (thoward) [reply themes/default/content/blog/next-level-iac-package-ecosystems/index.md:2]: yeah, I mostly went with this title for the alliteration and the general idea that package ecosystems are a value add to Pulumi that doesn't exist in other solution.

File: themes/default/content/blog/next-level-iac-package-ecosystems/index.md (hunk 1 above)
Comment (thoward) [reply themes/default/content/blog/next-level-iac-package-ecosystems/index.md:84]: I agree the CSS should be tweaked cause that yellow is a little intense.

File: themes/default/content/blog/next-level-iac-package-ecosystems/index.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+// Export the name of the bucket
+export const bucketName = bucket.id;
+
+// Export the URL of the file in the S3 bucket
+export const objectUrl = pulumi.interpolate`s3://${bucket.id}/${bucketObject.key}`;
+
+```
+
+
+Well that was pretty straightforward, wasn't it? Trying to do this in anther tool might take all day to figure out the right way to implement it, but in Pulumi it worked exactly like you would expect it to, in no time at all.
```
[the remark this answers is not in the document]
Comment (thoward) [reply themes/default/content/blog/next-level-iac-package-ecosystems/index.md]: I went ahead and paraphrased your comment here in another paragraph.

**Pull request merged**